docs/operations/containerized_data_importer.md

```diff
@@ -101,16 +101,140 @@ uploading the file to the `cdi-uploadproxy`.
 
     virtctl image-upload dv cirros-vm-disk --size=500Mi --image-path=/home/mhenriks/images/cirros-0.4.0-x86_64-disk.img --uploadproxy-url=<url to upload proxy service>
 
+### Addressing Certificate Issues when Uploading
```

Comment:
How about adding an H2 heading at L103 so the H3s show up in the Left Frame Table of Contents.

Maybe something like ...
`Extend virtctl functionality` or something like that.

Reply:
How about just moving "Addressing Certificate Issues when Uploading" to be an H2 and the other two to be H3 instead of H4?